yt_dlp/options.py

```diff
@@ -408,6 +408,14 @@ def _alias_callback(option, opt_str, value, parser, opts, nargs):
         help=(
             'Location of the main configuration file; either the path to the config or its containing directory '
             '("-" for stdin). Can be used multiple times and inside other configuration files'))
+    general.add_option(
+        '--plugin-dirs',
+        dest='plugin_dirs', metavar='PATH', action='append',
+        help=(
+            'Path to append to the default list of directories yt-dlp uses to search for plugins. '
+            'Note that this currently only works for extractor plugins; '
+            'postprocessor plugins can only be loaded from the default plugin directories. This '
+            'option can be used multiple times to add multiple directories'))
```

Comment:
I liked Grub's suggestion better
```suggestion
        help=(
            'Path to an additional directory to search for plugins. '
            'Note that this currently only works for extractor plugins; '
            'postprocessor plugins can only be loaded from the default plugin directories. '
            'This option can be used multiple times to add multiple directories'))
```